config/locales/client.en.yml

```diff
@@ -424,7 +424,31 @@ en:
 
   profile_page:
     messages:
-      updated: 'Profile updated'
+      updated: 'Your profile has been updated'
+      picture_changed: 'Your profile picture has been updated'
+      password_changed: 'Your password has been updated'
+      deactivated: 'Successfully deactivated your account'
+    change_picture:
```

Comment:
James the rule I've been following and enjoying is that the translations follow the same naming as BEM. Which reduces nesting. So this would be change_picture_form.title etc. Following the same rule everywhere is going to make it really easy to break the translations up into individual files.. just like the css you'll find change_picutre_form keys in the change_picture_form.yml file.

Can you please change these to match the name of the block they belong to?